m2m-transfers/V3/REPWRITERSPECS/BANNO.M2MTRANSFERS.V3.POW

```diff
@@ -2317,20 +2325,20 @@ PROCEDURE READCONFIGFILESETTINGS
          PARAMINSTINDIVLIMIT=MONEY(VALUE(PARAMVALUE))
         END
 
-
 [* Cross-Account Access
 *]
        ELSE IF PARAMNAME="XA" THEN
         DO
          IF CHARACTERSEARCH("1YESTRUE",UPPERCASE(PARAMVALUE))>0 THEN
           PARAMCROSSACCTACCESS=TRUE
         END
-[* Transfer From Type
+[* Transfer From Type - defaults to SHARE(S)
 *]
        ELSE IF PARAMNAME="XT" AND
                PARAMVALUE<>"" THEN
         DO
-         IF UPPERCASE(PARAMVALUE)="LOAN" THEN
+         IF UPPERCASE(PARAMVALUE)="LOAN" OR
```

Comment:
@rrobison44 Since the default (per the CFG Letter file) is "SHARES", I feel that falling back on that value as the default is the "safest" bet to make.  If the CFG Letter file setting is not either (specifically) "LOAN", "LOANS" or "BOTH" then the program falls back on to "SHARES". The alternative would be to add specific tests for "SHARE" and "SHARES" but then we'd have to add in additional error handling in the event it wasn't ONLY set up as "SHARE", "SHARES", "LOAN", "LOANS" or "BOTH".

Reply:
Ok.  I guess I was thinking you could still just fall back to shares if the value is not in the list: "SHARE", "SHARES", "LOAN", "LOANS" or "BOTH".  I think that is what you are doing now, anyways.  A value of "SOMETHING" would fall back to shares now.  My only point is that it is more clear if you actually have the "SHARE"/"SHARES" checks in the config read.  But it is good as is...in any case.

Reply:
@rrobison44 - Would you be thinking of something like this:

`
         IF UPPERCASE(PARAMVALUE)="LOAN" OR
            UPPERCASE(PARAMVALUE)="LOANS" THEN
          PARAMNTSLTYPE=NTSLTYPELOAN
         ELSE IF UPPERCASE(PARAMVALUE)="SHARE" OR
              UPPERCASE(PARAMVALUE)="SHARES" THEN
           PARAMNTSLTYPE=NTSLTYPESHARE
         ELSE IF UPPERCASE(PARAMVALUE)="BOTH" THEN
          PARAMNTSLTYPE=NTSLTYPESHARELOAN
         ELSE
          PARAMNTSLTYPE=NTSLTYPESHARE
`

Reply:
@rrobison44 Requested change committed and pushed.